src/myFunctions/sendResetPassword.ts

```diff
@@ -0,0 +1,34 @@
+import uuid from 'uuid';
+import ejs from 'ejs';
+import emailTemplateResetPassword from './emailTemplateResetPassword';
+import { sendEmail } from './sendEmail';
+
+const TOKEN_TIMEOUT = 60 * 60 * 1000; // 1 hour
+
+const server_domain = process.env.SERVER_DOMAIN;
+
+const uuidv4 = uuid.v4;
+
+export const sendResetPassword = async (user: any, email: string) => {
```

Comment:
Maybe change the `any` to `IUser` if you're bothered lol

Reply:
Will look into this in a future PR